modules/bmo-performing-a-live-update-to-the-hostfirmwarecomponents-resource.adoc

```diff
@@ -44,34 +45,35 @@ $ oc patch hostfirmwarecomponents <hostname> --type merge -p \ <1>
 You can also use `oc edit <hostname> hostfirmwarecomponents -n openshift-machine-api` to update the resource.
 ====
 
-. Cordon the host by running the following command:
+. Cordon and drain the host by running the following command:
 +
 [source,terminal]
 ----
-$ oc cordon <hostname>
+$ oc cordon <node_name> <1>
 ----
-+
-The Bare Metal Operator (BMO) updates the `operationalStatus` parameter to `servicing`.
+<1> Replace `<node_name>` with the name of the node. The Bare Metal Operator (BMO) updates the `operationalStatus` parameter to `servicing`.
```

Comment:
<1> Replace `<node_name>` with the name of the node. The Bare Metal Operator (BMO) updates the `Operational Status` of the associated `BareMetalHost` parameter to `servicing`.

(Clarifying to make sure users don't look for OperationalStatus in node instead of BareMetalHost
[devscripts@host15 ~]$ oc describe bmh/ostest-worker-1 | grep -i perational
Operational Status:  OK )